app/models/document.rb

```diff
@@ -308,6 +308,15 @@ def self.slug
     title.parameterize.pluralize
   end
 
+  # Downcase but maintaining initials
+  def self.title_for_new_document
+    title.split(" ").map { |p| p.upcase == p ? p : p.downcase }.join(" ")
+  end
+
+  def self.title_starts_with_initials?
+    title[0...2].upcase == title[0...2]
+  end
```

Comment:
It doesn't look like this method is used anywhere? I assume it was a first attempt and not used in implementation.

Reply:
Oh, yes - I thought I'd taken that out.

Reply:
Fixed.